pkg/frontend/http/trace/server.go

```diff
@@ -197,8 +209,9 @@ func (server *server) findTrace(metric *requestMetric, serviceName string, query
 		ServiceName:   serviceName,
 		OperationName: cluster,
 		Tags:          tags,
-		StartTimeMin:  timestamp.Truncate(time.Minute * 30),
-		StartTimeMax:  timestamp.Truncate(time.Minute * 30).Add(time.Minute * 30),
+		StartTimeMin:  startTimestamp,
+		StartTimeMax:  endTimestamp,
+		NumTraces:     1000,
```

Comment:
If we only expect one result, would `1000` be unnecessary here?

Reply:
jaeger storage list will search for different TraceSources, so the raw traces from backend is not one, but we will merger it into once trace, so it should be set to len(KnownTraceSources)?

Reply:
what about queries that span over multiple halfhours? do we need to multiply by `(end - start) / 30min`?